src/scalper_utils.ts

```diff
@@ -445,3 +446,17 @@ export async function getJupiterPrice(
   const midPrice = (buyPrice + sellPrice) / 2;
   return midPrice;
 }
+
+export function setPriorityFee(
+  tx: Transaction
+) {
+  const modifyComputeUnits = ComputeBudgetProgram.setComputeUnitLimit({ 
```

Comment:
Do we need this many CU? Setting this too big will get you deprioritized

Reply:
Just referencing https://solanacookbook.com/references/basic-transactions.html#how-to-change-compute-budget-fee-priority-for-a-transaction

Todo to refine this logic, we should try to make it dynamic based on demand per market. ie. BONK was having issues this week but SOL fine